README.md

```diff
@@ -4,38 +4,75 @@
 
 - Recognize what scope is
 - Recognize local scope
+- Recognize scopes overlap
+- Recognize global scope
 
 ## Introduction
 
 Now that we've talked about methods, we will also discuss another very important
 concept -- scopes! Scope defines where in a program a _variable_ is accessible.
-A variable is a name that Ruby associates with a particular object. These
-objects are locations in memory that hold any data to be used by the program. It
-is important to understand scope, not just in Ruby, but in any programming
-language. Understanding this concept well can help developers avoid errors, like
-_variables_ not being defined and incorrect variable assignments, that can cause
-confusion and frustration.
+A variable is a name that Ruby associates with a particular value: For example
+`dog = "Poodle"` or `age = 32`. These objects are hold any information we want
+to save and reuse. Understanding this concept well can help developers avoid
+errors and debug _extremely_ sneaky scope bugs.
 
 ## Recognize What Scope Is
 
-Think of method scope like a hotel room. There are no windows to other rooms
-(you cannot see what's going on in other rooms). Inside of this room, we can see
-the variables. These variables are "known" and "visible" inside of the room.
-Other variables in another room are not known in or from this particular room,
-but are known and visible in those other rooms.
+Say you go to a Mexican restaurant for tacos for lunch. You go in and sit down.
+A waiter comes along, takes your order and disappears. You wait...
 
-Every time there is a method call (let's say that we've invited in some other
-unregistered guests), when it enters the room, it's now in a "new" scope. 
+And wait...
+
+And wait...
+
+You decide to flag down the waitstaff and ask "Hey, I've been waiting a really
+long time, do you know where my order is?"
+
+Within the context of it being lunch time in this taco restaurant, and you have
+created a relationship with the waiter, the waitstaff knows that "order for
+person at table #12" should be in their notepad, or should be in the kitchen,
+your order has meaning in that context. "Order for table #12" is a variable that
+"points to" your delicious tacos.
+
+Say you walked over to a store across the street and asked "Do you know where
+order for table #12 is?" In that context your variable doesn't make sense. This
+spot might be a pet grooming salon. They don't know about taco orders, they
+don't know about tables, and they certainly don't know about your recent order
+of tacos.
+
+In real life we have a sense of "scope" because understand what's an appropriate
+thing to ask about in a given context.
```

Comment:
need a transition. We've given a good r.w. model of a scope. something like...

"So scope is something we're familiar with. But in code scope functions much the same way."

The most **natural** place to start talking is general (i.e. global) something like

> When we assign a variable, we're defining it in the _global_ scope......even inside of a method we can access variables in that scope.

then it's natural to say...

> Now that' we've seen globally scoped variables are accessible in methods, is the reverse true? It is not. Variables defined in local scopes cannot be accessed in global scopes.